src/progressbar/progressbar.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('mgcrea.ngStrap.progressbar', [])
+.provider('$progressbar', function (){
+    var defaults = {
+      barType: '',
+      animate: function (){ return true;}
+    };
+
+    this.$get = function (){
+      return {
+        defaults: defaults
+      };
+    };
+  })
+  .directive('bsProgressbar', function ($progressbar){
+    return {
+      restrict: 'E',
+      transclude: true,
+      replace: true,
+      templateUrl: 'progressbar/progressbar.tpl.html',
+      scope:{
```

Comment:
I was using `&` for animate because I didn't know optional 2 way bindings existed, I've updated it to `=?` :). What's the advantage of using `scope:true`?